docs/VLSI/Tutorial.rst

```diff
@@ -82,7 +82,8 @@ To elaborate the ``Sha3RocketConfig`` (Rocket Chip w/ the accelerator) and set u
 
 .. code-block:: shell
 
-    make buildfile MACROCOMPILER_MODE='--mode synflops' CONFIG=Sha3RocketConfig VLSI_TOP=Sha3AccelwBB
+    export MACROCOMPILER_MODE='--mode synflops'
```

Comment:
I was told that it's generally not good practice to export these variables to the shell environment, hence why I had taken this out from a previous iteration of the docs. Perhaps it's better to make it clearer that one has 3 options (in order of best practice): 1) append to make command, 2) change the Makefile, 3) export the variable?

Reply:
Ok. I'll fix this in a different way then.